build/Makefile.global

```diff
@@ -160,5 +160,10 @@ prof-use:
 		fi; \
 	fi;
 
+$(srcdir)/main/gdb_inlined_script.c: $(srcdir)/tools/gdb/gen_gdb_inlined_script.php
```

Comment:
I haven't tested this yet, but this will most likely break phpize builds I think. The phpize creates the Makefile from this Makefile.global stub also. What we could here is add a new Makefile.frag into the main directory (probably). Or even better inject the Makefile code into the fragment from the configure.ac file, something similar to this:

```m4
cat >>Makefile.objects<<EOF
\$(srcdir)/main/gdb_inlined_script.c: \$(srcdir)/tools/gdb/gen_gdb_inlined_script.php
	@if test ! -z "\$(PHP)"; then \\
		\$(PHP) \$(srcdir)/tools/gdb/gen_gdb_inlined_script.php; \\
	fi;
EOF
```

Reply:
Ok, it wouldn't break phpize it would only add it to the Makefile for extensions without executing it...